src/main/java/org/cryptomator/ui/mainwindow/VaultListCellController.java

```diff
@@ -18,15 +21,17 @@ public class VaultListCellController implements FxController {
 
 	private final ObjectProperty<Vault> vault = new SimpleObjectProperty<>();
 	private final ObservableValue<FontAwesome5Icon> glyph;
+	private final BooleanBinding useCondensedMode;
 
 	private AutoAnimator spinAnimation;
 
 	/* FXML */
 	public FontAwesome5IconView vaultStateView;
 
 	@Inject
-	VaultListCellController() {
+	VaultListCellController(Settings settings) {
 		this.glyph = vault.flatMap(Vault::stateProperty).map(this::getGlyphForVaultState);
+		this.useCondensedMode = Bindings.createBooleanBinding(settings.useCondensedMode::get, settings.useCondensedMode);
```

Comment:
Don't create a [Binding](https://openjfx.io/javadoc/22/javafx.base/javafx/beans/binding/Binding.html) here, since we do not calculate anyhting. Just store the settings.useCondensedMode directly. If we don't need writing the value, you can use as instance type [ObservableValue](https://openjfx.io/javadoc/22/javafx.base/javafx/beans/value/ObservableValue.html).